### PR TITLE
[BXMSPROD-1976] enable backporting from all branches

### PR DESCRIPTION
**Pull request by lampajr**
Enable PR backporting from all branches

**JIRA**: 

https://issues.redhat.com/browse/BXMSPROD-1976

*Referenced Pull Requests*:

- https://github.com/kiegroup/kogito-pipelines/pull/903
- https://github.com/kiegroup/kogito-runtimes/pull/2842
- https://github.com/kiegroup/kogito-examples/pull/1594
- https://github.com/kiegroup/kogito-apps/pull/1647
- https://github.com/kiegroup/drools/pull/5050
- https://github.com/kiegroup/optaplanner/pull/2660
- https://github.com/kiegroup/optaplanner-quickstarts/pull/529
- https://github.com/kiegroup/optaweb-vehicle-routing/pull/853
- https://github.com/kiegroup/optaplanner-website/pull/574
- https://github.com/kiegroup/kogito-serverless-operator/pull/82
- https://github.com/kiegroup/kogito-docs/pull/306
- https://github.com/kiegroup/kogito-images/pull/1455
- https://github.com/kiegroup/kogito-operator/pull/1402

### CODE REVIEW

**Review by Ginxo — APPROVED** [review]
LGTM

File: .github/workflows/pr-backporting.yml [hunk 1]
```diff
@@ -32,7 +30,7 @@ jobs:
     strategy:
       matrix: 
         target-branch: ${{ fromJSON(needs.compute-targets.outputs.target-branches) }}
-      fail-fast: true
+      fail-fast: false
```
Comment (mareknovotny) [inline .github/workflows/pr-backporting.yml:33]: why the change here?

Comment (lampajr) [reply .github/workflows/pr-backporting.yml:33]: If set to `true` and you are going to backport to different branches.. as soon as one fails all the others will be canceled (even if they could work).

Setting this to `false` we isolate those different backportings such that some can fail (maybe due to conflicts) but some other might work.. depending on the backporting branch.

**Review by radtriste — APPROVED** [review]

**Comment by radtriste** [comment]
missing repositories ?
- kogito-images
- kogito-operator
- kogito-serverless-operator
- kogito-docs
- optaplanner-website

**Comment by radtriste** [comment]
please also link optaweb and optaplanner-quickstarts

**Comment by lampajr** [comment]
> missing repositories ?
> 
> * kogito-images
> * kogito-operator
> * kogito-serverless-operator
> * kogito-docs
> * optaplanner-website

yeah I missed them, doing it

**Comment by lampajr** [comment]
@radtriste added remaining ones

**Comment by radtriste** [comment]
all approved. feel free to merge them :)

**Comment by mareknovotny** [comment]
please also announce this change on kogito/bsig ML we can have backporting from any branch not only main as a original PR
